### ISSUE TITLE
buildah info should probably indicate runtime

### ISSUE THREAD

**Issue by edsantiago** [containers/buildah#1964]
The `buildah info` command should probably include something like podman's `OCIRuntime`. Right now it's not easy to figure out whether it's runc or crun:
```
# buildah run --help|grep runtime
      --runtime path               path to an alternate OCI runtime (default "crun")
      --runtime-flag strings       add global flags for the container runtime
```

buildah-1.12.0-0.59.dev.git1967973.fc32.x86_64

**Comment by m7mdkamal** [comment]
I can start working on this issue if there is no one working on it.

**Comment by TomSweeneyRedHat** [comment]
Thanks @m7mdkamal !  I've assigned you to it.

**Comment by rhatdan** [comment]
@m7mdkamal Feel free to grab issues, we can always use more help.

**Comment by TomSweeneyRedHat** [comment]
Fixed by #1968
